supervisor/shared/filesystem.c

```diff
@@ -137,11 +141,19 @@ bool filesystem_init(bool create_allowed, bool force_create) {
         // https://specifications.freedesktop.org/trash-spec/trashspec-latest.html
         #endif
 
+        #if CIRCUITPY_SDCARDIO || CIRCUITPY_SDIOIO
+        res = f_mkdir(&vfs_fat->fatfs, "/sd");
+        #if CIRCUITPY_FULL_BUILD
+        MAKE_FILE_WITH_OPTIONAL_CONTENTS(&vfs_fat->fatfs, "/sd/placeholder.txt",
+            "This placeholder file allows mounting an SD card at /sd\n");
```

Comment:
The file itself isn't required so I'd say something different.

```suggestion
            "SD cards are mounted at /sd and will shadow this file from Python. USB will not see the SD card contents here.\n");
```

Reply:
done!